EncodingStreamingHighAvailability/HighAvailability.JobOutputStatus/JobOutputStatusFunction.cs

```diff
@@ -0,0 +1,86 @@
+namespace HighAvailability.JobOutputStatus
+{
+    using HighAvailability.Helpers;
+    using HighAvailability.Interfaces;
+    using Microsoft.Azure.EventGrid;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.EventGrid;
+    using Microsoft.Extensions.Logging;
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Implements job output status Azure function. It is triggered by EventGrid events
+    /// </summary>
+    public class JobOutputStatusFunction
+    {
+        /// <summary>
+        /// Service to process job output status events
+        /// </summary>
+        private IJobOutputStatusService jobOutputStatusService { get; set; }
+
+        /// <summary>
+        /// Service to to parse EvetGridEvents
+        /// </summary>
+        private IEventGridService eventGridService { get; set; }
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="jobOutputStatusService">Service to process job output status events</param>
+        /// <param name="eventGridService">Service to to parse EvetGridEvents</param>
+        public JobOutputStatusFunction(IJobOutputStatusService jobOutputStatusService, IEventGridService eventGridService)
+        {
+            this.jobOutputStatusService = jobOutputStatusService ?? throw new ArgumentNullException(nameof(jobOutputStatusService));
+            this.eventGridService = eventGridService ?? throw new ArgumentNullException(nameof(eventGridService));
```

Comment:
Is ternary recommend .NET style now?

Reply:
I believe null-coalescing operator ?? is widely accepted pattern. Do you have any concerns using it?

Reply:
No, this is a common pattern.  I don't have concerns with it in our samples.